iphone/Classes/KrollBridge.m

```diff
@@ -1152,65 +1167,105 @@ - (TiModule *)loadNodeModules:(NSString *)path withDir:(NSString *)start withCon
 - (id)require:(KrollContext *)kroll path:(NSString *)path
 {
 	NSURL *oldURL = [self currentURL];
+	NSString *workingPath = [oldURL relativePath];
+	NSString *pathCacheKey;
```

Comment:
Should be mutable (`NSMutableString`), since line 1181 can override it's current value.